src/CommonITILObject.php

```diff
@@ -6891,6 +6891,14 @@ public function getTimelineItemtypes(): array
             'item'          => new ITILSolution(),
             'hide_in_menu'  => !$canadd_solution
         ];
+        $itemtypes['document'] = [
+            'type'          => 'Document_Item',
+            'class'         => Document_Item::class,
+            'icon'          => Document_Item::getIcon(),
+            'label'         => _x('button', 'Add a document'),
+            'template'      => 'components/itilobject/timeline/form_document_item.html.twig',
+            'item'          => new Document_Item(),
```

Comment:
Should not be displayed in menu if user has no right to add a document.
```suggestion
            'item'          => new Document_Item(),
            'hide_in_menu' => !$canadd_document,
```